content/post/NEW-ipfs-camp-recap.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2019-07-05
+url: 2019-07-05-ipfs-camp-recap
+tags: IPFS, IPFS Camp
+title: 2019 IPFS Camp Recap
+author: David Dias
+---
+
+Greetings InterPlanetary Community!
+
+The InterPlanetary Space Training Camp is over and now that we are all back to Earth, I've the joy of sharing the news and learnings with you.
+
+If you aren't familiar with the [InterPlanetary Space Training Camp (IPFS Camp for short), it was a 3 day Hacker retreat for the Builders of the DWeb](https://camp.ipfs.io/). Attendees travelled through a [space portal](https://twitter.com/ChrisPacia/status/1144291810874920967) to get to Space Training Station in another Galaxy (Star coordinates yet undisclosed). Throughout the Camp, attendees had the chance to participate in:
+
+- [**👩🏽‍🏫 Core & Elective Courses**](https://github.com/ipfs/camp#-core--elective-courses) - Specialized and focused trainings on building for the DWeb.
+- [**📃 Poster Projects**](https://github.com/ipfs/camp#-poster-projects) - Collaborate in Groups to understand, explain and present how multiple parts of the DWeb Stack work
+- [**🐋 Deep Dives**](https://github.com/ipfs/camp#-deep-dives) - Form small teams to understand, research and come up with solutions to one of the many open problems
+- [**⚡️ Lightning Talks**](https://github.com/ipfs/camp#%EF%B8%8F-lightning-talks) - Learn what is the latest from multiple projects in the IPFS ecosystem
+- [**🧬 Sci-Fi Fair**](https://github.com/ipfs/camp#-sci-fi-fair) - Get blown away by huge Sci-Fi experiments, soon to be seen in a Network close to you
+- [**🧩 Unconf**](https://github.com/ipfs/camp#-unconf) - Gather folks around shared topics of interest and discuss collaborations.
+
```

Comment:
![image](https://user-images.githubusercontent.com/618519/60735219-c1426780-9f52-11e9-92b7-f9b64eb9193c.png)
![image](https://user-images.githubusercontent.com/618519/60735264-e931cb00-9f52-11e9-8bae-c7f6d7e6ac83.png)
I'd love to see a little more content about the courses and lightning talks - do you think adding an image for each one and/or indexing on a few particularly notable talks would work?

Reply:
Let me think more about this, I've one idea.